mujoco_warp/_src/support_test.py

```diff
@@ -85,6 +100,91 @@ def test_make_put_data(self):
       if isinstance(val, wp.array):
         self.assertEqual(val.shape, getattr(d, attr).shape)
 
+  _CONTACTS = """
+    <mujoco>
+      <worldbody>
+        <body pos="0 0 0.55" euler="1 0 0">
+          <joint axis="1 0 0" type="free"/>
+          <geom fromto="-0.4 0 0 0.4 0 0" size="0.05" type="capsule" condim="6"/>
+        </body>
+        <body pos="0 0 0.5" euler="0 1 0">
+          <joint axis="1 0 0" type="free"/>
+          <geom fromto="-0.4 0 0 0.4 0 0" size="0.05" type="capsule" condim="3"/>
+        </body>
+        <body pos="0 0 0.445" euler="0 90 0">
+          <joint axis="1 0 0" type="free"/>
+          <geom fromto="-0.4 0 0 0.4 0 0" size="0.05" type="capsule" condim="1"/>
+        </body>
+      </worldbody>
+    </mujoco>
+  """
+
+  def test_contact_force(self):
+    mjm, mjd, m, d = _load_from_string(self._CONTACTS)
+
+    # map MJX contacts to MJ ones
+    def _find(g):
+      val = (g == mjd.contact.geom).sum(axis=1)
+      return np.where(val == 2)[0][0]
+
+    contact_id_map = {i: _find(mjd.contact.geom[i]) for i in range(mjd.ncon)}
```

Comment:
MJWarp contacts align with MuJoCo contacts, so this index search should be unnecessary.